README.md

```diff
@@ -1,169 +1,169 @@
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- CONTRIBUTING -->
-
-## 🤝 Contributing <a name="contributing"></a>
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
-
-If you like this project consider adding a ⭐️
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-- I would like to thank Microverse.
-- Original design idea by: Gregoire Vella on Behance.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1 align="center">Budget Monitor</h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget app ](#-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget app <a name="about-project"></a>
+
+This Budget App is a user-friendly web app that empowers individuals to take control of their personal finances. By monitoring all transactions in one place, users get an always up-to-date overview of spending - enabling better decisions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Tech-stack</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="">Rspec</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **register and log in, so that the data is private to them.**
+- **Categorize purchases to see exact amounts spent on groceries, eating out, shopping, bills etc.**
+- **Set custom budgets for each category and track over/under spending**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo & Presentation <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-app-stqp.onrender.com)

```

Comment:
Hey @Izebiafe, I wanted to thank you for fixing the error that was preventing the deployed app from opening. However, I noticed that when I try to access the app from the provided link, I get an error saying 'Not Found'. I have attached a video to show you what I mean. Can you please verify and provide a working link in the readme file. Wish you good luck!


https://github.com/Izebiafe/Budget-app/assets/55840999/0245ca41-3688-4047-bd85-965e933ad336